sunpriya/__init__.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask,jsonify,request,make_response
+import constants as constants
+from exception import CSSException
+app=Flask(__name__)
+@app.route(constants.API_PATH_PREFIX + constants.API_PATH_EMAIL,methods=['GET'])
+def get_email_id():
```

Comment:
Don't get the point of this function, why is this endpoint needed? GET email? Why?